x/eibc/types/demand_order.go

```diff
@@ -67,6 +71,17 @@ func (m *DemandOrder) Validate() error {
 	return nil
 }
 
+func (m *DemandOrder) GetEvents() []sdk.Attribute {
```

Comment:
events.go?

Reply:
also same as before as it's part of this struct and I find it cleaner to define those methods on structs hence keeping it as part of the struct file. 
if you think it's better to move to events lets talk maybe it is I just don't see the reason.